packages/reports/tests/unit/consumers/report/table-visualization-test.js

```diff
@@ -70,10 +70,32 @@ test('reorder metrics', function (assert) {
 
 
   Ember.run(() => {
-    this.subject().send(ReportActions.UPDATE_TABLE_COLUMN_ORDER, { currentModel: currentModel }, newColumns);
+    this.subject().send(ReportActions.UPDATE_TABLE_COLUMN_ORDER, { currentModel }, newColumns);
   });
 
   assert.deepEqual(get(currentModel, 'request.metrics').map(metric => get(metric, 'canonicalName')),
     ['b', 'a'],
     'updateColumnOrder updates the order of the metrics in the request metrics');
+});
+
+test('UPDATE_TABLE_COLUMN', function (assert) {
+  assert.expect(1);
+
+  let currentModel = {
+    request: {
+      metrics: arr()
+    },
+    visualization: {
+      type: 'table',
+      metadata: { columns: [{ field: 'a', value: 1 }, { field: 'b', value: 2 }] }
```

Comment:
yeah, but I feel it may not be necessary for the testing purpose?

Reply:
Are the column fields not actually being used? If they are being used, they should be objects to be consistent. If they aren't being used, why have them at all?

Reply:
@Xuefeng-Zhu Can you address this please

Reply:
@mattgraham1995 Since this is unit test, I think we should make sure that the test can check if the function is working as intended. Therefore, it may not matter much what the value stored in `field`?

Reply:
Then why is the field even there? I think we should either remove `field` entirely or have it be the correct shape

Reply:
`field` is used to find the old column to relace

Reply:
So then why is `field` not taking the shape it should? It takes two seconds to change this to an object

Reply:
@mattgraham1995 updated. Sorry for the long disucssion